addon/templates/components/nypr-o-header.hbs

```diff
@@ -1,5 +1,5 @@
 {{#if this.rules.leaderboard}}
-  {{yield (hash leaderboard=(component 'nypr-o-header/leaderboard'))}}
+  {{yield (hash leaderboard=(component 'nypr-o-header/leaderboard' aria-hidden='true'))}}
```

Comment:
I did a quick search thru the repo for other places we're using `attributeBindings`, and I saw that when aria attributes are bound to a component (that has a root tag), the value is bound in the component instead of the template.

Would you mind moving this binding to `leaderboard.js`?
```js
export default Component.extend({
...
  'aria-hidden': true,
...
});
```

Reply:
Sure, this is cleaner than including the tag in the template. Thanks!